tests/integration/test_feedback.py

```diff
@@ -139,3 +140,105 @@ def test_feedback_events_without_processing(
 
     userfeedback = envelope.items[0]
     assert userfeedback.type == "feedback"
+
+
+def test_feedback_envelope_with_attachment(
+    mini_sentry,
+    relay_with_processing,
+    feedback_consumer,
+    attachments_consumer,
+    use_feedback_topic,
+):
+    mini_sentry.add_basic_project_config(
+        42, extra={"config": {"features": ["organizations:user-feedback-ingest"]}}
+    )
+    # mini_sentry.set_global_config_option("feedback.ingest-topic.rollout-rate", 1.0)
+    # TODO: make and set a FF here
```

Comment:
Are the `TODO`s in this test part of the scope of this PR?

Reply:
Oh, that's weird, the changes to this file are all for another PR, not sure how I included them here. Will revert